packages/vscode-extension/src/debug/teamsfxDebugProvider.ts

```diff
@@ -39,12 +44,16 @@ export class TeamsfxDebugProvider implements vscode.DebugConfigurationProvider {
           debugConfiguration.timeout = 20000;
         }
 
-        const teamsAppId = await commonUtils.getLocalDebugTeamsAppId(
+        const debugConfig = await commonUtils.getDebugConfig(
           isLocalSideloadingConfiguration
         );
+
+        // Put env and appId in `debugConfiguration` so debug handlers can retrieve it and send telemetry
+        debugConfiguration.teamsfxEnv = debugConfig?.env;
```

Comment:
Is it officially support by vscode that `DebugConfiguration` can have additional fields and can be passed to any consumers?